e2e/sanity_test.go

```diff
@@ -62,5 +65,13 @@ func testSanity() {
 		_, _, err = kubectl("exec", "-n", "topolvm-system", "daemonset/topolvm-node", "--", "test", "-e", path)
 		return sanity.PathIsOther, err
 	}
+
+	if os.Getenv("TEST_THIN_DEVICECLASS") == "true" {
+		// csi.storage.k8s.io/fstype=xfs,topolvm.cybozu.com/device-class=sbvglvm
+		volParams := make(map[string]string)
+		volParams["csi.storage.k8s.io/fstype"] = "xfs"
+		volParams["topolvm.cybozu.com/device-class"] = "thin"
+		tc.TestVolumeParameters = volParams
```

Comment:
I'm afraid I could not find any documents about `TestVolumeParameters`.
I would be happy if you could share the document you referred to when you write that code.

Reply:
I found it in the csi-test code:

https://github.com/kubernetes-csi/csi-test/blob/862e81af7f4b61e965714f8c528ee8ca91780dd7/pkg/sanity/sanity.go#L53

This describes the various parameters that can be specified.